src/seat.c

```diff
@@ -768,6 +767,8 @@ session_stopped_cb (Session *session, Seat *seat)
     if (session == priv->next_session)
         g_clear_object (&priv->next_session);
     if (session == priv->session_to_activate)
+        /* We were waiting for this session, but it didn't start :( */
```

Comment:
Can you either move the comment before the if as before or add braces to make it clear the g_clear_object is part of the branch.

Reply:
Done.